packages/react/src/global/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { CssFunction } from '../types';
+import { createSetupError } from '../utils/error';
+
+interface GlobalProps {
+  style: CssFunction | CssFunction[];
```

Comment:
🤔 Should we use `style` or something else? It might confuse consumers because react in built components accept `style` props which translates to inline styles.

Emotion also uses `styles` instead of `style`. Should we use that or something else 🤔?

Not sure 😅

Reply:
Ah definitely 😄 should have been `styles`